SPECS/avahi/avahi.spec

```diff
@@ -405,6 +410,21 @@ exit 0
 %endif
 
 %changelog
+* Mon Dec 02 2024 Kanishk Bansal <kanbansal@microsoft.com> -0.8-2
```

Comment:
Can you please combine all of these comments into a single changelog entry? That is how we typically mention multiple patches with the same release number.

Reply:
done